tools/makeboards.py

```diff
@@ -274,9 +274,16 @@ def BuildGlobalMenuList():
     print("menu.boot2=Boot Stage 2")
     print("menu.wificountry=WiFi Region")
     print("menu.usbstack=USB Stack")
+    print("menu.espwifitype=ESP wifi type")
```

Comment:
Minor thing, but could it be `ESP WiFi Type` (capitalization diff) to match the existing menu formats?